### ISSUE TITLE
Reload button doesn't work

### ISSUE THREAD

**Issue by asdert2** [ytti/oxidized-web#186]
Hi!
Reload button doesn't work. Nothing happens when i press it. But for example Refresh button works.
I have noticed that Reload button works only several seconds after restarting oxidized process or oxidezed service (if running as service). While all my devices have blue status icon I can press Reload button and I can see "reloaded list of nodes". But after several seconds as soon as any of my devices became green status I cannot press Reload button anymore - nothing happens.
I'm running Ubuntu 18.04.2. Its a fresh system, just installed it for Oxidized a day ago.
By the way, oxidized works fine, I'm getting all my configs successfully.
So now I have to restart oxidized service if I change router.db file. But it's not very handy. Can somebody help please?

Here's my config file:
```
username: user
password: password
model: routeros
resolve_dns: true
interval: 3600
use_syslog: false
debug: false
threads: 30
timeout: 20
retries: 3
prompt: !ruby/regexp /^([\w.@-]+[#>]\s?)$/
rest: 192.168.1.10:8888
next_adds_job: false
vars: {}
groups: {}
models: {}
pid: "/home/oxidized/.config/oxidized/pid"
crash:
  directory: "/home/oxidized/.config/oxidized/crashes"
  hostnames: false
stats:
  history_size: 10
input:
  default: ssh
  debug: false
  ssh:
    secure: false
  ftp:
    passive: true
  utf8_encoded: true
output:
  default: git
  git:
    user: oxidized
    email: user@company.com
    repo: "/home/oxidized/.config/oxidized/devices.git"
source:
  default: csv
  csv:
    file: /home/oxidized/.config/oxidized/router.db
    delimiter: !ruby/regexp /:/
    map:
      name: 0
      ip: 1
      model: 2
      username: 3
      password: 4
    vars_map:
      ssh_port: 5
model_map:
  juniper: junos
  cisco: ios
```

**Comment by asdert2** [comment]
Just noticed that I can successfully reload router.db by manually going to http://192.168.1.10:8888/reload

**Comment by arddg** [comment]
Hi! Same problem here. Any ideas how to fix it?

**Comment by arddg** [comment]
This solution fixed the problem for me:
https://github.com/ytti/oxidized-web/issues/194

**Comment by wk** [comment]
Should be closed by #210, thanks for troubleshooting this!
